src/oceans/ui.jsx

```diff
@@ -37,6 +31,7 @@ import {
   faInfo,
   faTrash
 } from '@fortawesome/free-solid-svg-icons';
+import * as I18n from './i18n';
```

Comment:
will unit tests break if we don't call `I18n.init()` before we test a component that uses i18n strings? if they do, we should make this module mockable so that we aren't depending on it being initialized before every test

Reply:
hm I'm not sure but our unit tests are passing

Reply:
oh lol nevermind they're not now that I've merged master in. I'll definitely fix the failing test cases

Reply:
we talked offline, but commenting for future us:

i lied, we should actually just call `I18n.init()` in our tests because we have some unit tests that depend on accurate translations:

https://github.com/code-dot-org/ml-activities/blob/e144c663de0592cbcf3a312870d3ae6833bb0961/test/unit/oceans/ui.test.js#L270

and it would be more annoying to mock every return value from i18n than it is to just initialize it at the beginning of test suites that need it